lib/krikri/parsers/primo_parser.rb

```diff
@@ -0,0 +1,29 @@
+module Krikri
+  ##
+  # A Primo parser. Uses XML parser with a root path to match the
+  # metadata path.
+  # @see Krikri::XmlParser
+  class PrimoParser < XmlParser
+    include Krikri::OaiParserHeaders
```

Comment:
Do we need OaiParserHeaders? If so, we should document why, and maybe change the name of that module to just `ParserHeaders` or something.